packages/web/src/components/Prizes/PrizesList.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Prize } from '@hangar/shared';
+import { Flex, Heading, ListItem, Text, UnorderedList } from '@chakra-ui/react';
+import { colors } from '../../theme/colors';
+
+type PrizeCardProps = {
+  prize: Prize;
+};
+
+type PrizesListProps = {
+  prizes: Prize[];
+};
+
+const RANKING= [ '🥇', '🥈', '🥉' ];
+const BONUS = '✨';
+
+const PrizeCard: React.FC<PrizeCardProps> = ({ prize }) => {
+  const { name, description, position, isBonus } = prize;
+
+  return (
+    <ListItem background={colors.brandPrimaryDark}>
+      <Heading as="h2" size="md">
+        <Flex gap={2}>
+          <Text>
+            { isBonus // eslint-disable-line 
+              ? BONUS
+              : position < RANKING.length
+                ? RANKING[position]
+                : null
+            }
```

Comment:
This is a little cleaner, but also please don't nest ternary expressions; this eslint rule is a good one
```suggestion
            { isBonus ? BONUS : RANKING[position + 1] ?? BONUS }
```